### DISCUSSION TITLE
Merchant of record support

### DISCUSSION THREAD

**Discussion by christophberger** [buttondown/discussions#38] [Ideas]
Hi,

Does anyone use a merchant of record like Paddle or Lemon Squeezy (the MoR, not the Swedish band) for paid subscriptions? 
If so, how did you do the integration?

Background: Stripe isn't a merchant of record, hence I would have to register with local tax authorities in each country that collects tax from payments, once a certain transaction limit has been exceeded. Stripe lists more than 50 countries that collect taxes. I am not really looking forward to going through the hassles of registering with even five of those countries.

**Comment by jmduke** [comment]
Great question! (Also, TIL Lemon Squeezy is also a Swedish band.)

The most common integration path I've seen thus far is [connecting LS to Zapier and then connecting it to Buttondown](https://zapier.com/apps/lemon-squeezy/integrations). We could probably do more to make this a "cleaner" integration, though if we were to integrate with a MoR it'd be Paddle to give us a bit of hedge against Stripe (since LS owns Stripe).

**Reply by christophberger** [reply]
After the LS acquisition, I would expect that Stripe will eventually integrate MoR to its tax collection service, maybe even so that it remains transparent to the existing Stripe API. But who knows...

What also came to my mind: I could manage paid subscribers entirely outside Buttondown and generate different emails via the API. Would be a lot more effort though.

**Reply by pirj** [reply]
It is indeed possible to sell and acquire payments via Lemon Squeezy or alike, but how would it be possible to prevent people from subscribing directly with Buttondown? There still will be a kink to subscribe on the buttondown.com/<project-name> page, won’t there?
This creates another problem - some subscriptions payments are processed via MoR, and non issues with e.g cross-border VAT, while others directly via Buttondown, and those are hard.

Another scenario I can think of is to turn off the “paid subscriptions “ feature off on Buttondown. This would allow accepting payments via MoR exclusively.
But this allows everyone to subscribe for free on the Buttondown public project page.
Is there a way to hide the “Subscribe” form on the public product page, or hide the public product page entirely from the world?

**Reply by jmduke** [reply]
@pirj there sure is! This is what [private mode](https://docs.buttondown.com/private-mode#enabling-private-mode) is for.
